.github/workflows/preview-image.yml

```diff
@@ -5,16 +5,14 @@ on:
       - Tests
     types:
       - completed
-    branches:
-      - master
 
 env:
   DOCKER_REPO: redash
 
 jobs:
   build-skip-check:
     runs-on: ubuntu-22.04
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    if: ${{ github.event.workflow_run.conclusion == 'success' && (startsWith(github.ref, 'refs/tags/') || startsWith(github.ref, 'refs/heads/main')) }}
```

Comment:
`main` seems like the wrong word here, or am I misunderstanding what this is doing?

Reply:
oh, thanks, updated